frontend/src/views/admin/matching/position-list.tsx

```diff
@@ -0,0 +1,132 @@
+import React from "react";
+import classNames from "classnames";
+
+import { FaFilter } from "react-icons/fa";
+import { Form } from "react-bootstrap";
+
+import { PositionSummary } from "./types";
+import { round } from "../../../libs/utils";
+
+import "./styles.css";
+
+function PositionRow({
+    summary,
+    isFocus,
+    setSelectedPosition,
+}: {
+    summary: PositionSummary;
+    isFocus: boolean;
+    setSelectedPosition: Function;
+}) {
+    const targetHours = round(
+        summary.position.hours_per_assignment *
+            (summary.position.desired_num_assignments || 0),
+        2
+    );
+
+    let backgroundStyle = "";
+
+    switch (summary.filledStatus) {
+        case "over":
+            backgroundStyle = "rgba(0, 229, 243, 0.15)";
```

Comment:
No! Do not hardcode CSS. Use classes and actual CSS.

Reply:
Made changes to address this but there's still one part that's _kinda_ hardcode-y in order to get the right width for in-progress positions.